powerprofile/powerprofile.py

```diff
@@ -268,6 +268,15 @@ def min(self, magn):
         if self._check_magn_is_valid(magn):
             return self.curve[magn].min()
 
+    def avg(self, magn):
+        """
+        Returns avg value of given magnitude of the curve
+        :param magn: magnitude value
+        :return: avg magnitude value
+        """
+        if self._check_magn_is_valid(magn):
+            return self.curve[magn].agg({magn: 'mean'})[magn]
```

Comment:
Diria que es pot fer una mica més senzilla la sintaxi així, però el resultat segurament és el mateix, si es prova.
```suggestion
            return self.curve[magn].mean()
```

Reply:
Done